src/main/java/com/zufar/icedlatte/security/api/SecurityPrincipalProvider.java

```diff
@@ -16,11 +17,14 @@ public class SecurityPrincipalProvider {
     private final UserDtoConverter userDtoConverter;
 
     public UserDto get() {
-        UserEntity userEntity = (UserEntity) SecurityContextHolder
+        var auth = SecurityContextHolder
                 .getContext()
-                .getAuthentication()
-                .getPrincipal();
-         return userDtoConverter.toDto(userEntity);
+                .getAuthentication();
+        assert auth instanceof UsernamePasswordAuthenticationToken :
```

Comment:
looks not good
do we really need it?

Reply:
To me it's better, than `ClassCastException` in case of anonymous access, that would be thrown if developer forgot to add a new path to `SecurityConstants.java`

Reply:
Reverted in [c5d340b](https://github.com/Sunagatov/Iced-Latte/pull/217/commits/c5d340bf9bece2f311eb821361ed28fe71a89b22)